### DISCUSSION TITLE
more notes/details for the examples

### DISCUSSION THREAD

**Discussion by RaquisDawLn** [hyperium/hyper#2982] [Q&A]
Im new to rust and im wanting to use Hyper for a http client but i cant figure out how to make it print the body of the page i.e http://httpbin.org/ip will respond with {   "origin": "IP" }(in json format obv) but i want to print the it to console using println! instead of stdout etc

**Comment by seanmonstar** [comment]
There's a similar example for dealing with JSON: https://github.com/hyperium/hyper/blob/0.14.x/examples/client_json.rs
